lib/ansible/modules/cloud/azure/azure_rm_mysqlserver.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_mysqlserver
+version_added: "2.5"
+short_description: Manage MySQL Server instance
+description:
+    - Create, update and delete instance of MySQL Server
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    sku:
+        description:
+            - The SKU (pricing tier) of the server.
+        suboptions:
+            name:
+                description:
+                    - The name of the sku, typically, a letter + Number code, e.g. P3.
+            tier:
+                description:
+                    - "The tier of the particular SKU, e.g. Basic. Possible values include: 'Basic', 'Standard'"
+            capacity:
+                description:
+                    - "The scale up/out capacity, representing server's compute units."
+            size:
+                description:
+                    - The size code, to be interpreted by resource as appropriate.
+            family:
+                description:
+                    - The family of hardware.
+    location:
+        description:
+            - The location the resource resides in.
+    storage_mb:
+        description:
+            - The maximum storage allowed for a server.
+    version:
+        description:
+            - "Server version. Possible values include: '5.6', '5.7'"
+    ssl_enforcement:
+        description:
+            - "Enable ssl enforcement or not when connect to server. Possible values include: 'Enabled', 'Disabled'"
+    create_mode:
+        description:
+            - "Currently only 'Default' value supported"
+    admin_username:
+        description:
+            - "The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation)."
+    admin_password:
+        description:
+            - The password of the administrator login.
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Create (or update) MySQL Server
```

Comment:
can you use real values for this example, you've just copied the key name for the values.

Reply:
yes, updated